Assets/Scripts/DayNightCycle/ToggleDayNightController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Assets.Scripts.Services;
+using Assets.Scripts.Settings;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+namespace Assets.Scripts.DayNightCycle
+{
+    public class ToggleDayNightController : MonoBehaviour
+    {
+        public Color DefaultSkyColor;
+        public Color DefaultEquatorColor;
+        public Color DefaultHorizonColor;
+        public GameObject DayNightControllerPrefab = null;
+        public Toggle ToggleDayNight;
+        public Button PauseDayNightButton;
+        public GameObject DayNightController;
+        public GameObject MainLight;
+        public Text Label;
+        public Material skyBoxReset;
+        private string time;
+        private double seconds;
+
+        DayAndNightControl dayNightCycle;
+        public GameObject DefaultLightSet;
+        // Start is called before the first frame update
+        void Start()
+        {
+            DefaultLightSet = GameObject.Find("LightSet");
+            //These defaults are stored so that when the system is toggled off, all colour settings are set back to the scene defaults
+            DefaultSkyColor = RenderSettings.ambientSkyColor;
+            DefaultEquatorColor = RenderSettings.ambientEquatorColor;
+            DefaultHorizonColor = RenderSettings.ambientGroundColor;
+            ToggleDayNight.isOn = false;
+            ToggleDayNight.onValueChanged.AddListener(delegate {
+                ToggleValueChanged(ToggleDayNight);
+            });
+
+        }
+
+        void ToggleValueChanged(Toggle change)
+        {
+            if (PhotonNetwork.isMasterClient)
```

Comment:
We should only sync settings once we click the synchronize button

Reply:
This is again a UX decision, the user would have to anyway tap a synch button for this rpc to take effect, and this one only synchs once a toggle, so it would be no more RPC's being sent than it would with a synch button being pressed after changing the toggle value.

Reply:
Well even ignoring UX, sending it on ToggleValueChanged is the same as not having it, then tapping the synch button(which some users may tap more than once) after changing the toggle setting. By having it here it makes it easier to use and sends the same amount of RPC data anyway.